polkadot/node/core/backing/src/lib.rs

```diff
@@ -479,6 +488,12 @@ fn table_attested_to_backed(
 	}
 	vote_positions.sort_by_key(|(_orig, pos_in_group)| *pos_in_group);
 
+	if inject_core_index {
+		let core_index_to_inject: BitVec<u8, BitOrderLsb0> =
+			BitVec::from_vec(vec![core_index.0 as u8]);
+		validator_indices.extend(core_index_to_inject);
```

Comment:
Shouldn't we be using some nice `BackedCandidate` function to be introduced in the other PR for that?

Reply:
that's a good idea, will do that in the follow-up PR